fluent-bit/examples/override-fluentbit-http.yaml

```diff
@@ -0,0 +1,74 @@
+---
+#override.yaml
```

Comment:
name doesn't match the actual file name

Reply:
We should show only the relevant overrides, not the entire file.

Reply:
these are the relevant overrides, in the http integration, in order to use static name, you name to put the entire 'filters' section. 
and the 'inputs' and 'extraFiles' are there only because its an example so I show how to override these, 
should I take it off ? the inputs and extrafiles ?